src/mfakto.cpp

```diff
@@ -787,7 +787,16 @@ int load_kernels(cl_int *devnumber)
   }
   else
   {
+#if defined __APPLE__
+    snprintf(
```

Comment:
I was reluctant to just change `sprintf` to `snprintf` because I read that the latter does not behave consistently across different operating systems: https://stackoverflow.com/a/14503914/1285261

However, that post is from 11 years ago, so I don't know if that's still the case.

Reply:
I was reluctant to simply change `sprintf` to `snprintf` as I've read the latter does not behave consistently across different operating systems: https://stackoverflow.com/a/14503914/1285261

However, that post is from 11 years ago, so I don't know if that's still the case.